src/app/routes/_app/rulesets/$rulesetSlug/rulebooks/$rulebookSlug/index.module.css

```
@@ -0,0 +1,101 @@
+
+.pageLinks span {
+  color: var(--mantine-color-dimmed);
+  font-variant-numeric: tabular-nums;
+  text-align: end;
+}
+
+.readerDocument {
+  min-width: 0;
+}
+
+.readerDocument :global([data-rulebook-page]) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)